### ISSUE TITLE
+ sign in filename breaks vsix extensions

### ISSUE THREAD

**Issue by echalone** [browserify/resolve#242]
Hi, the "+" sign in the file name of the (since 1.19.0) new file ".github/workflows/node-4+.yml" breaks VSIX extensions which use this package since the path of the file is added as the asset type for each file in the extension.vsixmanifest manifest-file and a "+" sign is not allowed in there. The package will not be installable afterwards. Please just remove the + sign from the file name (name it "node-4p.yml" for example).
This extension is effected for example: https://github.com/dependency-check/azuredevops/

**Comment by ljharb** [comment]
What is VSIX?

Either way, that file isn't included in the published package, and nothing should ever be consuming the repo unless you're developing the project, so I'm not sure why it's causing a problem.

**Comment by echalone** [comment]
Hi, sorry, VSIX is the file extension for Visual Studio extensions, which are also used for Team Foundation/Azure DevOps extensions.
You're right, apperantly this file is no longer included in the newest package versions (but it was in older, which produced that problem) and all I had to do is make sure the newest version is retrieved. Thanks :)

I know that + is a valid character for file names and it's a bug that microsoft would not handle that correctly, but here we are ^^
I've also already filed this issue with microsoft's VSIX creation module.
